build.sbt

```diff
@@ -70,14 +70,10 @@ def common: Seq[Setting[_]] = releaseSettings ++ bintraySettings ++ evictionSett
   javacOptions in compile ++= Seq(
     "-encoding",
     "UTF-8",
-    "-source",
-    "1.8",
-    "-target",
-    "1.8",
     "-parameters",
     "-Xlint:unchecked",
     "-Xlint:deprecation"
-  ),
+  ) ++ akka.JavaVersion.sourceAndTarget(akka.CrossJava.Keys.fullJavaHomes.value("8")),
```

Comment:
Building in JDK11 needs the param `-bootclasspath` provided conditionally by `CrossJava`.